cumulus/client/cirrus-executor/src/tests.rs

```diff
@@ -315,19 +317,34 @@ async fn pallet_executor_unsigned_extrinsics_should_work() {
             panic!("The requested execution receipt for block {primary_number} does not exist")
         });
 
+        let bundle = Bundle {
+            header: BundleHeader {
+                primary_hash: ferdie.client.hash(primary_number).unwrap().unwrap(),
+                slot_number: (std::time::SystemTime::now()
+                    .duration_since(std::time::SystemTime::UNIX_EPOCH)
+                    .expect("Current time is always after unix epoch; qed")
+                    .as_millis()
+                    / 2000) as u64,
```

Comment:
I assume this can break if we change runtime parameters? Should we query runtime for these then?

Reply:
The test won't break as the slot number is now unchecked actually, any number could work, the reason I use the actual time here is to merely achieve some meaningful randomness. We can query the runtime once it's not the case.